src/impl/domain.ts

```diff
@@ -118,6 +118,28 @@ export function get(key: string): any {
   return store[key];
 }
 
+/**
+ * Retrieves all values that correspond to a given list of keys.
+ *
+ * Example:
+ *  const a = 1;
+ *  const b = 2;
+ *  const sum = a + b;
+ *  store.set({a, b, sum})
+ *  const results = store.getByKeys(['a', 'b', 'sum']); // {a: 1, b: 2, sum: 3}
+ *
+ * @param {Array<string>} keys
+ * @returns {*}
+ */
+export function getByKeys(keys: string[]): any {
```

Comment:
Generic function same as above. 
```suggestion
export function getByKeys<T>(keys: string[]): T[] {
```